Document/0x05a-Platform-Overview.md

```diff
@@ -1,37 +1,36 @@
-## Android Platform Overview
+# Android Platform Overview
 
-This chapter is going to introduce Android on the architecture point of view and will provide the reader with detailed information on it's security mechanisms. Then, it will describe the structure of an Android application and will emphasize on the Inter Process Communication (IPC) mechanisms. Last, the way Android applications are published is explained to the reader.
+This chapter introduces Android from an architecture point of view. It covers four key areas:
 
-The reader is invited to have a look at the official Android developer documentation<sup>[13]</sup> for more details on the Android platform. Even if numerous examples are given in this section, in no way this guide should be considered as the only reference on the topic.
+1. Android security architecture
+2. Android application structure
+3. Inter Process Communication (IPC)
+4. Android application publishing
 
-### Android Architecture and Security Mechanisms
+The reader is invited to have a look at the official Android developer documentation[13] for more details on the Android platform. 

```

Comment:
Maybe you could write this as an imperative mood instead of inviting the reader in passive voice? "Look at the official Android developer documentation..."